src/helpers/parsing-key-maps/isValidKey.js

```diff
@@ -1,7 +1,10 @@
-import isSpecialKey from './isSpecialKey';
+import isNonPrintableKeyName from './isNonPrintableKeyName';
+import Configuration from '../../lib/Configuration';
 
 function isValidKey(keyName) {
-  return isSpecialKey(keyName) || String.fromCharCode(keyName.charCodeAt(0)) === keyName;
+  return isNonPrintableKeyName(keyName) ||
+    String.fromCharCode(keyName.charCodeAt(0)) === keyName ||
+    Configuration.option('customKeyCodes')[keyName];
```

Comment:
⬜️🐛Fix this line - incorrect way of accepting custom keys (it's a map of key codes to key names and this lookup will always fail).